src/components/auth/Register.js

```diff
@@ -0,0 +1,27 @@
+import { auth } from "../../api/firebase";
+import { AuthForm } from "./AuthForm";
+import { createUserWithEmailAndPassword, signOut } from "firebase/auth";
+import { firebaseErrors } from "../../utils/firebaseErrors"
+import { getFormData } from "../../utils/getFormData";
+
+export const Register = () => {
+  const handleRegister = (e) => {
```

Comment:
Te funkcje wyglądają podobnie można je wydzielić do osobnego pliki np authHandlers i stamtąd importować np
import {handleRegister} 
i tutaj wrzucić to do funkcji handleClick = (e) =>  handleRegister(e)